server/core/src/main/scala/sttp/tapir/server/interceptor/decodefailure/DecodeFailureHandler.scala

```diff
@@ -257,14 +260,17 @@ object DefaultDecodeFailureHandler {
       }
     }
 
+    def pathMessage(path: List[FieldName]): Option[String] =
```

Comment:
is it useful to have both methods public?

Reply:
we still don't have to be backwards-compatible ;-) so I'm fine with leaving one method

Reply:
done